LibVLCSharp/Shared/Core.cs

```diff
@@ -111,6 +150,10 @@ internal static class Constants
         /// The name of the folder that contains the per-architecture folders
         /// </summary>
         internal const string LibrariesRepositoryFolderName = "libvlc";
+
+        internal const string Windows = "msvcrt";
```

Comment:
I don't think that those 3 constants have a good naming. Either you can create a static class named `StandardLibraryPath` and move them there, or you will need to prefix those constants